spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionServiceTests.java

```diff
@@ -1363,6 +1363,9 @@ private static void verifyTaskExistsInRepo(String taskName, String dsl,
 			TaskDefinitionRepository taskDefinitionRepository) {
 		TaskDefinition taskDefinition = taskDefinitionRepository.findById(taskName).get();
 		assertThat(taskDefinition.getName(), is(equalTo(taskName)));
+		if (taskDefinition.getProperties().containsKey("spring.cloud.dataflow.task.composed-task-element")) {
```

Comment:
Test is no longer needed due to the updates in the controller.  Will fix on merge.